### ISSUE TITLE
Broken disk undo

### ISSUE THREAD

**Issue by SBPrime** [SBPrime/AsyncWorldEdit#314]
```
//pos1 (at top of area)
(then few chunks away) //pos2 (y:3)
then i hold redstone torch in hand, and type: //replace hand minecraft:air
it is busy for a while, .. but since they are redstone_wall_torch it will not replace anything of course..
processed: 0 blocks
done...
console looking good at this point.
//undo
it does appear in-game it is instantly completed on undo, though there's nothing to undo ofcourse
and then console shows the error i posted.

Yeah, can reproduce what the admin did.

Code (Text):
>ver
[10:34:12] [Server thread/INFO]: This server is running CraftBukkit version git-Spigot-2f5d615-d07a78b (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)
[10:34:12] [Server thread/INFO]: Checking version, please wait...
[10:34:12] [Thread-161/INFO]: You are running the latest version
>ver WorldEdit
[10:34:15] [Server thread/INFO]: WorldEdit version 7.1.0;8e55131
>ver AsyncWorldEdit
[10:34:18] [Server thread/INFO]: AsyncWorldEdit version 3.7.2
[10:34:18] [Server thread/INFO]: This plugin attaches into WorldEdit and makes it asynchronous.
[10:34:18] [Server thread/INFO]: Website: https://github.com/SBPrime/AsyncWorldEdit/wiki
[10:34:18] [Server thread/INFO]: Author: SBPrime
>
```

After thise steps the //undo command says that file was not found.

**Comment by mrfloris** [comment]
Replying so i get update notifications. Thank you for posting it here, sorry for not doing so in teh first place. btw, java 11.

**Comment by SBPrime** [comment]
Fixed in 3.7.3
